build.py

```diff
@@ -1102,6 +1102,12 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu,
     if 'onnxruntime' in backends:
         df += '''
 ENV LD_LIBRARY_PATH /opt/tritonserver/backends/onnxruntime:${LD_LIBRARY_PATH}
+'''
+
+    # Necessary for libtorch.so to find correct HPCX libraries
+    if enable_gpu and ('pytorch' in backends):
+        df += '''
+ENV LD_LIBRARY_PATH /opt/hpcx/ucx/lib/:${LD_LIBRARY_PATH}
```

Comment:
I was wondering whether opencv is ok with this order. I think this means that opencv would pick up the newer hpcx version.